### PR TITLE
changes to mifare plus code

### PR DESCRIPTION
**Pull request by drhatson**
Have happy, healthy and fun Halloween!

### CODE REVIEW

**Comment by merlokk** [comment]
Cool)

File: client/cmdhfmfp.c [hunk 1]
```diff
@@ -439,7 +448,7 @@ int CmdHFMFPRdbl(const char *cmd) {
 	bool verbose = arg_get_lit(1);
 	int blocksCount = arg_get_int_def(2, 1);
 	bool keyB = arg_get_lit(3);
-	int plain = arg_get_lit(4) | true;
+	int plain = arg_get_lit(4);
```
Comment (pwpiwi) [inline client/cmdhfmfp.c]: Now that we have a "Plain" flag, the help text should be without the "in plain mode"

File: client/cmdhfmfp.c [hunk 2]
```diff
@@ -467,6 +476,10 @@ int CmdHFMFPRdbl(const char *cmd) {
 		return 1;
 	}
 	
+	if (blocksCount > 1 && mfIsSectorTrailer(blockn)) {
```
Comment (pwpiwi) [inline client/cmdhfmfp.c]: Why do we need to warn here?

File: client/cmdhfmfp.c [hunk 3]
```diff
@@ -545,7 +558,7 @@ int CmdHFMFPRdsc(const char *cmd) {
 	
 	bool verbose = arg_get_lit(1);
 	bool keyB = arg_get_lit(2);
-	bool plain = arg_get_lit(3) | true;
+	bool plain = arg_get_lit(3);
```
Comment (pwpiwi) [inline client/cmdhfmfp.c]: Same as above . Remove "in plain mode" from the helptext.

File: client/cmdhfmfp.c [hunk 4]
```diff
@@ -467,6 +476,10 @@ int CmdHFMFPRdbl(const char *cmd) {
 		return 1;
 	}
 	
+	if (blocksCount > 1 && mfIsSectorTrailer(blockn)) {
+		PrintAndLog("WARNING: trailer!");
```
[the remark this answers is not in the document]
Comment (drhatson) [reply client/cmdhfmfp.c]: card don't read trailer if blocksCount > 1

File: client/cmdhfmfp.c (hunk 2 above)
Comment (drhatson) [reply client/cmdhfmfp.c]: same

**Pull request merged**